cmd/prometheus/main.go

```diff
@@ -206,6 +206,10 @@ func reloadConfig(filename string, rls ...Reloadable) (success bool) {
 	conf, err := config.LoadFile(filename)
 	if err != nil {
 		log.Errorf("Couldn't load configuration (-config.file=%s): %v", filename, err)
+		// TODO(julius): Remove this notice when releasing 0.17.0 or 0.18.0.
+		if err.Error() == "unknown fields in global config: labels" {
+			log.Errorf("NOTE: The 'labels' setting in the global configuration section has been renamed to 'external_labels'. Please update your configuration file accordingly.")
```

Comment:
Addressed that. I didn't link to the specific release in the CHANGELOG.md, as we are removing RC changelog entries after a final release and thus the link would break with 0.16.0.